pkg/controller/restore-operator/http.go

```diff
@@ -39,27 +39,27 @@ func (r *Restore) startHTTP() {
 	panic(http.ListenAndServe(listenAddr, nil))
 }
 
-// serveBackup parses incoming request url of the form /backup/<cluster-name>
-// get the etcd cluster name.
+// serveBackup parses incoming request url of the form /backup/<restore-name>
+// get the etcd restore name.
 // Then it returns the etcd cluster backup snapshot to the caller.
 func (r *Restore) serveBackup(w http.ResponseWriter, req *http.Request) {
-	clusterName := string(req.URL.Path[len(backupHTTPPath):])
-	if len(clusterName) == 0 {
-		http.Error(w, "cluster is not specified", http.StatusBadRequest)
+	restoreName := string(req.URL.Path[len(backupHTTPPath):])
+	if len(restoreName) == 0 {
+		http.Error(w, "restore name is not specified", http.StatusBadRequest)
 		return
 	}
-	v, ok := r.restoreCRs.Load(clusterName)
+	v, ok := r.restoreCRs.Load(restoreName)
 	if !ok {
-		http.Error(w, fmt.Sprintf("cluster %v backup server not found", clusterName), http.StatusInternalServerError)
+		http.Error(w, fmt.Sprintf("restore %v backup server not found", restoreName), http.StatusInternalServerError)
 		return
 	}
 
-	logrus.Infof("serving backup for cluster %v", clusterName)
+	logrus.Infof("serving backup for cluster %v", restoreName)
```

Comment:
maybe change this log to something else since we are not reasoning about cluster anymore.